src/api.js

```diff
@@ -4,8 +4,14 @@ const newsApi = axios.create({
   baseURL: 'https://majids-backend-api-project.onrender.com/api'
 });
 
-export const fetchArticles = (topic, sortBy, limit = 50) => {
-  return newsApi.get('/articles', { params: { topic , sortBy, limit} })
+export const fetchArticles = (topic, sortBy) => {
+  const sortDesc = ['votes', 'comment_count', 'created_at'];
+  return newsApi.get('/articles', { params: { 
+    topic , 
+    sort_by: sortBy, 
+    limit: 50, 
+    order: sortDesc.includes(sortBy) ? 'desc' : 'asc'
```

Comment:
im not completely sure what this does.

Reply:
> im not completely sure what this does.

Thanks for taking a look. I will delete the comments. About this bit of code, order: sortDesc.includes(sortBy) ? 'desc' : 'asc', I just wanted to sort the articles by descending order if the sort filter is numeric, otherwise the sort order is alphabetical. Later, I will be adding a way to let the user choose the sort order.